fossology/fossology
# feat(edit-user): Let user can define default folder and use the defau…

…e Default Folder at upload pages and browse page



## Description

Users (including non-admin) can set newly added 'Default Folder' at Edit User Account page.
At browse page, the 'Default Folder' is used as 'Root' of the tree on navigation screen.
At upload pages(from File, URL and etc.), the 'Default Folder' is pre-set so that user do not need to navigate through all the folders on FOSSology.

Users can select the 'Default Folder' among folders that he/she is accessible.(which is set by the Admins)

### Changes

1) 'default_folder_fk' column is added at 'users' table
2) A drop down menu for 'default folder' is added on Edit User Account page.
3) Upload pages and Browse page shows from the 'default folder'

## How to test

1) Set 'Default Folder' at Edit User Account page.
2) Check Browse page and Upload pages if the 'Default Folder' is reflected.
